setup.cfg

```diff
@@ -29,7 +29,7 @@ packages = find:
 install_requires =
     bqplot>=0.11.6
     ipywidgets>=7.6.0,<8
-    pandas>=0.25.0
+    pandas<=1.3.5
```

Comment:
What's wrong with newer pandas? This will bound to make the installation hard for some requiring newer pandas; if it's a bad pandas release better to exclude just that one :thinking:

Reply:
The issue is here:
https://github.com/bloomberg/ipydatagrid/blob/c0fd0fc97c21daa157ccae42268fc82b0157aacd/tests/test_datagrid.py#L133

Since versions > 1.3.5, pandas is now updating the version number in its to_dict() method. The unpinned installation version of pandas for older versions of Python such as 3.6 and 3.7 seems to differ from the default one for Python >= 3.8. So the older Python tests fail in the CI. We should definitely look into it and see if there is a better way!